src/stores/AppStore.ts

```diff
@@ -1,7 +1,51 @@
 import { EventEmitter } from 'events';
+import { SnackbarPosition } from '../components/AppBar/NotificationSnackbar';
+import { CalendarEvent, CourseEvent } from '../components/Calendar/CourseCalendarEvent';
+import { RepeatingCustomEvent } from '../components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog';
+import { AACourse, AASection } from '../peterportal.types';
 import { calendarizeCourseEvents, calendarizeCustomEvents, calendarizeFinals } from './calenderizeHelpers';
 
+export interface UserData {
+    addedCourses: AppStoreCourse[]
+    scheduleNames: string[]
+    customEvents: RepeatingCustomEvent[]
+}
+
+export interface AppStoreCourse {
+    color: string
+    courseComment: string
+    courseNumber: string //i.e. 122a
+    courseTitle: string
+    deptCode: string
+    prerequisiteLink: string
+    scheduleIndices: number[]
+    section: AASection
+    // sectionCode: string
```

Comment:
Why is this commented?

Reply:
I probably did that because I wasn't sure if it was safe to remove, but now that I'm done with everything and nothing broke because it's commented out, I'm just gonna remove it.